spec/controllers/concerns/idv_step_concern_spec.rb

```diff
@@ -12,44 +12,6 @@ class StepController < ApplicationController
     end
   end
 
-  describe '#confirm_idv_session_started' do
```

Comment:
These tests don't make sense in the post-FSM world as described in the pull request description. I added a test to the phone controller spec to test the equivalent logic. The review controller already had a matching test.